packages/yoga/src/Text/native/__snapshots__/Text.test.jsx.snap

```diff
@@ -278,7 +353,7 @@ exports[`<Text /> Snapshots should match snapshot with Text light 1`] = `
           "color": "#231B22",
           "fontFamily": "Rubik",
           "fontSize": 48,
-          "fontWeight": "300",
+          "fontWeight": "500",
```

Comment:
This and the others components with the `light` prop should keep the weight of 300.